packages/shared/components/atoms/MenuButton.svelte

```diff
@@ -0,0 +1,58 @@
+<script>
+    import { Icon, Text } from 'shared/components'
+
+    export let icon
+    export let title
+    export let subtitle = ''
+    export let onClick
+    export let first = false
+    export let last = false
+    export let selected = false
+    export let disabled = false
```

Comment:
This is used because by default we want some props to be falsey, and only when they include them it is true.

i.e. 
<Menu Button /> is not disabled
<Menu Button disabled /> is disabled

Reply:
Done